rdt-app/src/main/java/io/ona/rdt_app/application/RDTApplication.java

```diff
@@ -90,4 +94,25 @@ public void onTerminate() {
         SyncStatusBroadcastReceiver.destroy(this);
         super.onTerminate();
     }
+
+    public static CommonFtsObject createCommonFtsObject() {
+        if (commonFtsObject == null) {
+            commonFtsObject = new CommonFtsObject(getFtsTables());
+            commonFtsObject.updateSearchFields(PATIENTS, getFtsSearchFields());
+            commonFtsObject.updateSortFields(PATIENTS, getFtsSortFields());
+        }
+        return commonFtsObject;
+    }
+
+    private static String[] getFtsTables() {
+        return new String[]{PATIENTS};
+    }
+
+    private static String[] getFtsSearchFields() {
+        return new String[]{Constants.DBConstants.BASE_ENTITY_ID, Constants.DBConstants.ID, Constants.DBConstants.NAME};
+    }
+
+    private static String[] getFtsSortFields() {
+       return new String[]{Constants.DBConstants.BASE_ENTITY_ID, Constants.DBConstants.ID, Constants.DBConstants.NAME};
```

Comment:
Sort with `Name`, `BASE_ENTITY_ID` and `ID` does not make sense since its based on UUIDs

Reply:
Changed to using names alone.